### PR TITLE
fix(compiler): invalid json schema generated for structs with `Json`

### PR DESCRIPTION
**Pull request by hasanaburayyan**
Closes: https://github.com/winglang/wing/issues/5961

## Summary
- Remove `jsonschema` library in place for `Ajv` which is used also in the platform parameters, it supports latest Json schema revisions, where `jsonschema` does not
- Enable type unions in json schema validation
- Changes generated json schemas of json objects from: `type: "object"` to use type unions for any valid json -> `type: ["object", "string", "number", "boolean", "array"]

## Checklist

- [ ] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [ ] Description explains motivation and solution
- [ ] Tests added (always)
- [ ] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

File: examples/tests/sdk_tests/std/struct.test.w
Comment (hasanaburayyan) [inline examples/tests/sdk_tests/std/struct.test.w:1]: silly me I forgot to add a test here to show the issue is fixed 😂 incoming

**Review by Chriscbr — APPROVED** [review]
🚀

**Comment by staycoolcall911** [comment]
That's great, thanks for the quick fix!

**Review by eladb — COMMENTED** [review]
Awesome

**Pull request merged**